File: tacas2020-benchmarks/vaphor/compare.java
class vapexp{
	static int i, x, SIZE, rv;
	static int a[], b[];

static void main()
{
	SIZE=Support.random();

	rv = 1;
	i = 0;
	while ( i < SIZE ) {
		if ( a[i] != b[i] ) {
			rv = 0;
		}
		i = i+1;
	}

	x=0;
        while(x<SIZE)
        {
		assert( rv == 0 || a[x] == b[x] );
		x = x + 1;
        }
}
}

File: tacas2020-benchmarks/booster/compare.c
#define SIZE
int main( ) {
	int i;
	int a[SIZE];
	int b[SIZE];
	int rv;

	rv = 1;
	i = 0;
	while ( i < SIZE ) {
		if ( a[i] != b[i] ) {
			rv = 0;
		}
		i = i+1;
	}

	int x;
	for ( x = 0 ; x < SIZE ; x++ ) {
		assert( rv == 0 || a[x] == b[x]  );
	}
	return 0;
}
